spec/playload/parser/helpers/create_classifications_spec.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+RSpec.describe Parser::Helpers::CreateClassifications do
+  let(:result)       { described_class.new(new_params).call(call_params) }
+  let(:data)         { File.read('spec/support/sample_data.json') }
+  let(:game_details) { JSON.parse(data)['GameInfo'][0] }
```

Comment:
Could maybe think about moving this into the spec helper to save us declaring it in many different spec files

Reply:
Hmm I'm not sure about that one to be honest. They don't appear in every spec and I like the idea that you can see the setup without changing files. You care enough to counter?

Reply:
No